changed formatting

elikreuz opened the pull request:
@GreatEmerald

GreatEmerald commented on index.Rmd:
```diff
@@ -144,8 +150,10 @@ Before continuing, please check if this step was successful.
+<center>
 <img src="./images/gdal.png" alt="gdal structure" width = "90%">
 <figcaption>GDAL class structure, adapted from Garrard, 2016, Geoprocessing with Python.</figcaption>
+<center>
```
This is not a closing tag now, since you forgot a `/`.

GreatEmerald reviewed (changes requested):
Thanks, one minor point below.

elikreuz commented:
@GreatEmerald I changed the links to the WCS because they were not working, and also changed how the building were loaded in with GeoPandas and requests, since the previous exercise they used a different method (and again the link for buildings was giving errors.

GreatEmerald commented:
Thanks!
